Pilot/PilotFile.swift

```diff
@@ -39,13 +37,13 @@ extension PilotFile {
   }
 
   var description: String {
-    return "File{name=\(name), size=\(size), writeTime=\(writeTime), directory=\(directory)}"
+    return "File{name=\(name), size=\(size), directory=\(directory)}"
   }
 
 }
 
-enum FileType {
-  case Photo
-  case Video
-  case Document
+enum FileType: String {
```

Comment:
Not sure that this enum fits well in this swift file.. maybe move it somewhere else?